src/NHSD.BuyingCatalogue.Solutions.Persistence/Repositories/SolutionCapabilityRepository.cs

```diff
@@ -12,51 +12,57 @@ namespace NHSD.BuyingCatalogue.Solutions.Persistence.Repositories
 {
     public sealed class SolutionCapabilityRepository : ISolutionCapabilityRepository
     {
-        private readonly IDbConnector _dbConnector;
-
         private const int PassedFullCapabilityStatus = 1;
 
-        public SolutionCapabilityRepository(IDbConnector dbConnector) =>
-            _dbConnector = dbConnector.ThrowIfNull(nameof(dbConnector));
-
-        private const string sql = @"SELECT Capability.Id as CapabilityId,
+        private const string Sql = @"SELECT Capability.Id as CapabilityId,
                                         Capability.Name as CapabilityName,
-                                        Capability.Description as CapabilityDescription
+                                        Capability.Description as CapabilityDescription,
+                                        Capability.Version as CapabilityVersion,
+                                        Capability.SourceUrl as CapabilitySourceUrl
                                 FROM SolutionCapability
                                      INNER JOIN Capability ON SolutionCapability.CapabilityId = Capability.Id
-                                WHERE SolutionCapability.SolutionId = @solutionId
+                                     INNER JOIN SolutionCapabilityStatus ON SolutionCapabilityStatus.Id = SolutionCapability.StatusId
+                                WHERE SolutionCapability.SolutionId = @solutionId AND SolutionCapabilityStatus.Pass = 1
```

Comment:
Rather than hard code the value '1', do you think it would be a good idea to supply that value as a parameter using a constant to give it context?